LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceCache.cs

```diff
@@ -240,7 +244,10 @@ public bool TryGetByDevEui(string devEUI, [MaybeNullWhen(returnValue: false)] ou
         {
             lock (this.syncLock)
             {
-                _ = this.euiCache.TryGetValue(devEUI, out device);
+                if (this.euiCache.TryGetValue(devEUI, out device))
+                {
+                    this.deviceCacheHits?.Add(1);
```

Comment:
[nit] we could also track it together in `TrackCacheStats`, where we essentially also keep track of the statistics for the internal implementation